roles/gateway_configure_db_replication/tasks/add_slave_user.yml

```diff
@@ -0,0 +1,21 @@
+---
+# This task creates a MySQL slave user on a Gateway.
+
+- name: Add slave user for {{ slave_node }} to MySQL on {{ master_node }} (may take several minutes)
+  no_log: "{{ no_log_flag }}" # don't log decrypted passwords in responses
+  expect:
+    command: sshpass -p '{{ gateway_root_password }}' ssh {{ ansible_ssh_extra_args }} root@{{ master_node }} /opt/SecureSpan/Appliance/bin/add_slave_user.sh
+    timeout: 600
```

Comment:
1 of my GWs took 9 min for this step to finish... not sure if that is typical.
Suggestion for appropriate timeout is most welcome :)